src/main/java/seedu/address/model/person/Patient.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.person;
+
+import java.util.Set;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.affiliation.Affiliation;
+
+/**
+ * Represents a Patient in the address book.
+ */
+public class Patient extends Person {
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Patient(Name name, Phone phone, Email email, Address address, Set<Affiliation> affiliations) {
+        super(name, phone, email, address, affiliations);
```

Comment:
And perhaps we can have a constructor for Patient/Doctor class that do not require setting the role if it is clearer; i.e. just super(name, phone, email, "Patient", affiliations)

Reply:
Will do that as well 👌